server/src/main/java/io/crate/statistics/ReservoirSampler.java

```diff
@@ -213,8 +243,16 @@ private Samples getSamples(List<Reference> columns,
             int readerId = FetchId.decodeReaderId(fetchId);
             DocIdToRow docIdToRow = docIdToRowsFunctionPerReader.get(readerId);
             Object[] row = docIdToRow.apply(FetchId.decodeDocId(fetchId));
+
             try {
-                rowAccounting.accountForAndMaybeBreak(row);
```

Comment:
Is it correct that we have removed this call? Seems that now we don't have a circuitbreaker exception if accounting exceeds memory limits.

Reply:
sorry sorry, I somehow missed it. Thx!